lib/browser/init.ts

```diff
@@ -192,10 +194,13 @@ app.on('window-all-closed', () => {
   }
 })
 
-const { setDefaultApplicationMenu } = require('@electron/internal/browser/default-menu')
-
-// Create default menu.
-app.once('ready', setDefaultApplicationMenu)
+Promise.all([
+  import('@electron/internal/browser/default-menu'),
```

Comment:
here too

Reply:
Same as above, and the async nature here actually works well with `app.whenReady`